example/exporter/kube-state-metrics/servicemonitor.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: "monitoring.coreos.com/v1alpha1"
+kind: "ServiceMonitor"
+metadata:
+  name: "kube-state-metrics"
```

Comment:
this can be generalized to be something like `k8s-apps-http`, so signal to the user reading this you don't actually need to create a `ServiceMonitor` per `Service`, but rather create a convention for `Service`s, so all `Service`s can be discovered by one `ServiceMonitor`. In reality, probably more than one `ServiceMonitor` will be necessary, but the idea is to keep the number of `ServiceMonitor`s very small compared to the number of `Service`s.